release-notes/RC1-RC2_Upgrade.md

```diff
@@ -0,0 +1,52 @@
+# RC1 to RC2 Upgrade Roadmap
+
+Due to schedule changes for the .NET Core all-up, there was a rename of the tooling from "RC2" to "RC2 - Preview 1". This rename impacted all of the packages as well as installers for .NET Core and .NET Core tools. The impact of this change for the users is in that all previous versions of .NET Core and any tooling **must be** removed from the machine in order to properly install and use RC2 release.
+
+**What about RC1 bits?**
+
+This does not impact RC1 release. Since the tooling and the way you write applications has changed, having DNX (or many of them) installed should not be a problem. However, one caveat is what tooling you are using for what project. For RC2 projects you may get wrong results and/or errors if you try to restore dependencies using `dnu restore`. Please use the appropriate tooling. You can see what are the new commands in the DNX to CLI migration document listed below.
+
+For updating your code, you can check these two documents:
+
+* [Migrating from DNX to CLI](http://dotnet.github.io/docs/core-concepts/dnx-migration.html)
+* [Updating from ASP.NET Core RC1 to RC2]()
```

Comment:
We need to add this link prior to merging. @danroth27 needs to give us the link when the UE person is done with writing the doc.